Automattic/wp-calypso
# Add the Social Premium plan and enhanced publishing feature

#### Proposed Changes

This change introduces the new enhanced sharing feature for social and adds the premium yearly and monthly plans to Calypso. It also fixes the naming convention for the 1000 shares feature.

#### Testing Instructions

Needs 1203313664893883-as-1203207028774795 merged before testing



* Head to http://calypso.localhost:3000/checkout/jetpack/jetpack_social_advanced_yearly to add Jetpack Social Premium Yearly to your shopping cart
* Verify that two variants appear (yearly and monthly)
* Verify that you can switch between the variants
 * Verify that you can complete the purchase using credits or the store sandbox.


#### Pre-merge Checklist






Related to #
